schemas/annotations/create.json

```diff
@@ -0,0 +1,174 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "required": ["text"],
+  "properties": {
+    "target": { "type": "string", "description": "The target URL of the document" },
+    "sourcedb": { "type": "string", "description": "Document source database" },
+    "sourceid": { "type": "string", "description": "Document source ID" },
+    "text": { "type": "string", "description": "Document text" },
+    "project": { "type": "string", "description": "Project name" },
+    "denotations": { "$ref": "#/definitions/denotations" },
+    "attributes": { "$ref": "#/definitions/attributes" },
+    "relations": { "$ref": "#/definitions/relations" },
+    "blocks": { "$ref": "#/definitions/blocks" },
+    "typesettings": { "$ref": "#/definitions/typesettings" },
+    "config": { "$ref": "#/definitions/config" },
+    "project_id": { "type": "string", "description": "Project ID" },
```

Comment:
おそらく、ユーザーに`project_id`をそのまま見せることはないと思っています。

Reply:
`project`と`project_id`があるのは自分も変だと思ったのですが、TextAEでアノテーション投稿をするとリクエストに両方ついていたので書いています。

`"project"=>"test-project1"`, ` "project_id"=>"test-project1"`
![image](https://github.com/user-attachments/assets/3aa0d747-5038-4847-8f63-8f6275bf2e08)

project_idはPubDictionariesではプロジェクト名としているようです。
https://github.com/pubannotation/pubdictionaries/blob/62b2bd1c08937ff185a9a31112943423abf62a4b/app/models/dictionary.rb#L149

AnnotationsController#createでも両方使用している様なので、違和感はありますが必要そうです・・
```
##########
# determine the project
##########
project = begin
	if unsafe_params.has_key? :project_id
		# if a project is specified in the path, it should exists and accessible. Otherwise an exception should be thrown.
		_project = Project.editable(current_user).find_by! name: unsafe_params[:project_id]
	else
		# if a project is specified in the annotation, better to use it
		_project = Project.editable(current_user).find_by_name(unsafe_params[:project]) if unsafe_params.has_key? :project

		# return _project if present. Otherwise return the default project of the current user
		_project || _project = current_user.default_project
	end
```